src/pages/index.svelte

```diff
@@ -3,27 +3,40 @@
     import { flip } from 'svelte/animate'
 	import Player from '../components/player.svelte'
 	import AddPlayer from '../components/addPlayer.svelte'
-	import Button from '@smui/button'
+	import Button , {Label} from '@smui/button'
 	import Card from '@smui/card'
-	import Paper, {Title, Subtitle, Content} from '@smui/paper'
+	import Paper, {Subtitle} from '@smui/paper'
+	import Dialog, {Title, Content, Actions} from '@smui/dialog'
 	import { currPlayer, players, removePlayer } from "../playerService.js"
 
     metatags.title = 'Arcade with routify'
     metatags.description = 'Play all your favourite games in one spot'
 	
+	let dialog
+	let dialogMessage
 	let name = ''
 	let showCtrl = true
 	let height = [1, 1, 1]
 	$: toggle = true
 
+	const remove = name => {
+		try {
+			removePlayer(name)
+		} catch(e) {
+			dialogMessage = e
+			dialog.open()
```

Comment:
Did not see a way to pass e into dialog.open so created dialogMessage instead of typing the message again in the element. This will also allow dialog to be recycled if another need arises.

Reply:
works for now!